QUICK_START.md

```
@@ -0,0 +1,162 @@
+1. Click "Edit Profile"
+2. Make some changes
+3. Click "Cancel"
+4. ✅ Changes are discarded
+
+## 🔧 Field Validation Rules
+
+| Field | Format | Example |
+|-------|--------|---------|
+| Email | valid@domain.com | john@example.com |
+| Username | 3-30 chars, a-z, 0-9, _, - | john_doe123 |
+| Phone | +[country][number] | +15551234567 |
```

markdownlint-cli2 (0.21.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "number"

(MD052, reference-links-images)